core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -710,7 +713,9 @@ map_processor_fold(#kz_binding{binding_parts=BParts
                   ,_Routing
                   ,RoutingParts
                   ) ->
-    case matches(BParts, RoutingParts) of
+    case matches(BParts, RoutingParts)
+        orelse matches(RoutingParts, BParts)
```

Comment:
I agree with creating a "copy with `orelse` added" even though as I see it that means lots of code duplication. I think it requires kazoo_bindings:map/N+1 (+1 = routes matched). A new module is overkill & baaa code duplication.

I disagree with multiple help arities however: it was first written this way but it makes it impossible to get good error report on bad input: in [Category, Action] if Action is wrong you have no way to reply "action is wrong" because it could also mean "category is wrong" too.